app/helpers/work_packages_helper.rb

```diff
@@ -472,9 +514,15 @@ def work_package_form_status_attribute(form, work_package, locals = {})
     new_statuses = work_package.new_statuses_allowed_to(locals[:user])
 
     field = if new_statuses.any?
-              form.select(:status_id, (new_statuses.map { |p| [p.name, p.id] }), required: true)
+              work_package_form_field do
+                form.select(:status_id,
+                            new_statuses.map { |p| [p.name, p.id] },
+                            required: true)
+              end
             elsif work_package.status
-              form.label(:status) + work_package.status.name
+              work_package_form_field do
+                form.label(:status, class: "form--label") + wrap_element_in_container(work_package.status.name)
```

Comment:
Line is too long. [111/99]<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.